### ISSUE TITLE
Script "/usr/share/ntopng/scripts/lua/pro/report.lua" validation error

### ISSUE THREAD

**Issue by simonemainardi** [ntop/ntopng#985]
```
Script "/usr/share/ntopng/scripts/lua/pro/report.lua" returned an error:

/usr/share/ntopng/scripts/lua/modules/http_lint.lua:857: [LINT] _GET["zoom"] = "true" parameter error: Validation error


```

**Comment by emanuele-f** [comment]
Fixed with 7f48b1b4ad55c1ade4323cf76ceed5a0f768f4dd

**Comment by antonakv** [comment]
I have this issue again:

```
Internal server error

/usr/share/ntopng/scripts/lua/modules/http_lint.lua:2150: [LINT] Validation error: Unknown key toggle_date_type [value: middle_endian]: missing validation perhaps?_
```
```
$ ntopng --version
Version:	5.1.210829 [Enterprise/Professional/Embedded build]
GIT rev:	dev:cce12ef8bdba7f8e146ef5d7d33a369cb6638670:20210829
Pro rev:	r4202
Built on:	Raspbian GNU/Linux 10 (buster)
System Id:	xxxxxxxxxxxxxD2--OL
Platform:	armv7l
Edition:	Enterprise Embedded
```

It happens on http://192.168.x.x:3001/lua/admin/prefs.lua?tab=gui
- Authorise in ntopng
- Open http://192.168.x.x:3001/lua/admin/prefs.lua?tab=gui
- Click on Dark button
- Click Save

http://192.168.x.x:3001/lua/http_status_code.lua?message=internal_error&referer=192.168.x.x%3A3001%2Flua%2Fadmin%2Fprefs.lua%3Ftab%3Dgui&error_message=%2Fusr%2Fshare%2Fntopng%2Fscripts%2Flua%2Fmodules%2Fhttp_lint.lua%3A2150%3A%20%5BLINT%5D%20Validation%20error%3A%20Unknown%20key%20toggle_date_type%20%5Bvalue%3A%20middle_endian%5D%3A%20missing%20validation%20perhaps%3F%0A
